api/swagger.yaml

```diff
@@ -2228,6 +2230,37 @@ definitions:
       - "disconnected"
     example: "ready"
 
+  ManagerStatus:
+    description: |
+      ManagerStatus represents the status of a manager.
+
+      It provides the current status of a node's manager component, if the node
+      is a manager.
+    x-nullable: true
+    type: "object"
+    properties:
+      Leader:
+        type: "boolean"
+        default: false
+        example: true
+      Reachability:
+        $ref: "#/definitions/Reachability"
+      Addr:
+        description: |
+          Addr provides the IP address and port at which the manager is
+          reachable. As configured through `--advertise-addr`.
```

Comment:
s/As configured through/To configure, use the `--advertise-addr` flag

(or something like that)